source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -29,8 +29,42 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+- Most search functionality included in the core platform now supports Meilisearch as an alternative to Elasticsearch. For instances deployed using Tutor, only Meilisearch is supported, and Tutor will automatically provision Meilisearch and create the indexes during the upgrade.
+  - After upgrading, you need to run `manage.py cms reindex_studio --experimental --incremental` to populate the new Studio Search index with your existing content (courseware/libraries). Studio search results will be incomplete until this command has finished.
```

Comment:
```suggestion
   - After upgrading, you need to run ``manage.py cms reindex_studio --experimental --incremental`` to populate the new Studio Search index with your existing content (courseware/libraries). Studio search results will be incomplete until this command has finished.
```

Sorry, I suck at RST.

Reply:
No need to apologize. ReStructuredText is a festering carbuncle of a markup language and I hate it with flaming hate-e-os. :)

Reply:
Ugh, I wonder how much more documentation we'd have if we let people write it in Markdown...

Reply:
Oh come on, it's not that bad!